frontend/src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -15,11 +13,19 @@ const PopupWithForm = (props) => {
           }}
           className="popup__block"
         >
+ <form
+              className="popup__form"
+              onSubmit={props.onSubmit}
+              action="#"
+              method="POST"
+             // name={"signup"}
+            >
           <button onClick={props.onClose} className="popup__close-btn"></button>
```

Comment:
`            <button onClick={props.onClose} className="popup__close-btn"></button>` is duplicated 3 times. You have it inside `PopupWithForm` already